lib/nerves_init_gadget/application.ex

```diff
@@ -11,11 +11,12 @@ defmodule Nerves.InitGadget.Application do
 
     # Define workers and child supervisors to be supervised
     children = [
+      worker(Nerves.InitGadget.GadgetDevices, [merged_opts]),
```

Comment:
we should probably put `restart: :transient` here to stop from flailing in the event of failures.

Reply:
Do you mean [`:temporary`](https://hexdocs.pm/elixir/Supervisor.html#module-restart-values-restart), so that it just tries to run once and then gives up even if it fails? That would probably be fine so that at least the system comes up, for example, if they're using an current-generation system that doesn't have ConfigFS enabled.